examples/DB/Sqlite3/test_plan.py

```diff
@@ -110,11 +112,14 @@ def main(plan):
     :return: Testplan result object.
     :rtype:  ``testplan.base.TestplanResult``
     """
+    my_db_path = os.path.join(tempfile.gettempdir(), "new_db_name")
```

Comment:
Consider using an context mgr here so that the tmp dir is removed after plan run?

Reply:
`gettempdir()` is the temporary directory defined by the OS, it shouldn't be removed.